python/composio/tools/toolset.py

```diff
@@ -977,6 +978,47 @@ def get_entity(self, id: t.Optional[str] = None) -> Entity:
         """Get entity object for given ID."""
         return self.client.get_entity(id=id or self.entity_id)
 
+    def get_expected_params(
+        self,
+        app: t.Optional[AppType] = None,
+        auth_scheme: t.Optional[
+            t.Literal[
+                "OAUTH2",
+                "OAUTH1",
+                "API_KEY",
+                "BASIC",
```

Comment:
In the `get_expected_params` method, it might be better to use an Enum for the `auth_scheme` parameter instead of a string literal. This would provide better type safety and autocomplete support.